RandomUsersTest/app/src/test/java/com/skaravatskiy/randomusers/ViewModelUsersListTestPositive.kt

```diff
@@ -0,0 +1,118 @@
+package com.skaravatskiy.randomusers
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule
+import android.arch.lifecycle.Observer
+import com.google.gson.Gson
+import com.skaravatskiy.randomusers.data.model.common.Mapper
+import com.skaravatskiy.randomusers.data.model.database.User
+import com.skaravatskiy.randomusers.data.model.poko.UsersResponse
+import com.skaravatskiy.randomusers.data.repository.Repository
+import com.skaravatskiy.randomusers.ui.userslist.Sort
+import com.skaravatskiy.randomusers.viewmodel.UserListViewModel
+import io.reactivex.Observable
+import io.reactivex.android.plugins.RxAndroidPlugins
+import io.reactivex.schedulers.Schedulers
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.mockito.ArgumentMatchers
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.MockitoAnnotations
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import android.arch.lifecycle.LiveData as LiveData1
+
+@RunWith(JUnit4::class)
+class ViewModelUsersListTestPositive {
```

Comment:
We no need to create separate files for positive and negative test cases here as well